docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -32,6 +35,14 @@ Community-maintained tools are available to help detect which .NET Framework ver
 
 For information about detecting the installed updates for each version of .NET Framework, see [How to: Determine which .NET Framework updates are installed](how-to-determine-which-net-framework-updates-are-installed.md).
 
+## Determine which .NET version an app is running on
+
+You can use the <xref:System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription?displayProperty=nameWithType> property to query for which .NET implementation your app is running on. If the app is running on .NET Framework, the output will be similar to:
```

Comment:
Heading says "version", text says "implementation" -- it looks like "implementation" is a more accurate word to describe the purpose of the section. But if that's the case it might be worth showing .NET Core and .NET 5 examples too.